ArcGISRuntimeSDKQt_CppSamples/Scenes/ExploreScenesInFlyoverAR/ExploreScenesInFlyoverAR.cpp

```diff
@@ -0,0 +1,116 @@
+// [WriteFile Name=ExploreScenesInFlyoverAR, Category=Scenes]
+// [Legal]
+// Copyright 2019 Esri.
```

Comment:
2020 for new files.

Reply:
Thanks for catching that. New year always catches me by surprise.

Reply:
Need to update the samples template as well as they reference 2019 still.